### PR TITLE
generate images to cheat mlp network for mnist

### LINKED ISSUE #6511: How to optimize data layer in Paddle
This question is originally asked by a Baidu user. He wants to cheat the pre-trained neural network by just optimizing data layer.

In the previous Paddle, the gradient of the data layer is not calculated since it is not necessary for optimizing parameters.

In the PaddlePaddle Fuild, we are not separate data/parameter/activation at all. Every variable in a neural network can be optimized.

I will give an example of cheating a pre-trained neural network.

### PR DESCRIPTION
**Pull request by reyoung**
Fix #6511

### CODE REVIEW

File: python/paddle/v2/fluid/tests/demo/cheat_mnist.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            append_batch_size=False)
+        x.stop_gradient = False
+        x.persistable = True
+        label = fluid.layers.data(
+            name='label', shape=[1, 1], dtype='int64', append_batch_size=False)
+        label.persistable = True
+
+    cheat_program = data_program.clone()
+
+    with fluid.program_guard(cheat_program, cheat_init_program):
```
Comment (Canpio) [inline python/paddle/v2/fluid/tests/demo/cheat_mnist.py]: Why do we need to write cheat_net's `x` and `train_net` in two `with` statements? Seems that they are both in the `cheat_program`.

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献代码。由于Paddle V1/V2版本已不再维护，相关代码也已从develop分支上删除，因此关闭您的PR，欢迎您向Paddle最新版-Fluid贡献代码。
Thanks for contributing to PaddlePaddle! Since V1/V2 will not be maintained anymore, and related codes have been deleted from develop branch as well, we close this PR. Welcome to contribute to Fluid——the latest version of PaddlePaddle.
